RELEASE.rst

```diff
@@ -0,0 +1,3 @@
+RELEASE_TYPE: patch
+
+This release fixes :func:`builds(callable) <hypothesis.strategies.builds>` so that ``target`` can be used as a keyword argument for passing values to the target. The target itself can still be specified as a keyword argument, but that behavior is now deprecated. The target should be provided as the first positional argument.
```

Comment:
Our convention for strategies is to use the default link text for all but decorators, so just 

    :func:`~hypothesis.strategies.builds`

Wrap lines at 79 characters - the old convention is still useful for diffs etc., and we enforce it for the code (see the `lint` job on Travis).